Cargo.toml

```diff
@@ -27,17 +27,17 @@ name = "cargo-outdated"
 
 [dependencies]
 anyhow = "1.0"
-cargo = "0.76.0"
-env_logger = "0.10.0"
-git2-curl = "0.19.0"
+cargo = "0.82.0"
```

Comment:
This version of cargo depends on `orion` crates v0.17.7, for which MSRV is rust 1.80.1 (you can check it by running `cargo msrv`)

Maybe you should update the MSVR in Cargo.toml
```toml
rust-version = "1.80.1" # MSRV
```

But, LGTM

Reply:
Fixed, thanks!